stdlib/ostruct.rb

```diff
@@ -60,10 +66,52 @@ def to_n
   def hash
     @table.hash
   end
+  
+  def table
```

Comment:
This is a problem if anyone defines a field named `:table`, use an internal special field called `$$table`.